Framework7/features/feature_definitions/task_feature/task.feature

```diff
@@ -0,0 +1,13 @@
+@Task
+Feature:Create task
+  Scenario Outline: Create task in a story with only description
+    Given I have set a connection to pivotal tracker API service
+    And I have <project_id> from  Pivotal Tracker project the project
```

Comment:
The sentence should be improved if not it will be displayed of this way in the report:
I have 123456 from  Pivotal Tracker project the project